requirements.txt

```diff
@@ -1,6 +1,7 @@
 Django==3.1
 django-allow-cidr==0.3.1
 django-filter==2.3.0
+django-smart-selects @ git+https://github.com/jazzband/django-smart-selects.git@60022d061be96eacbad21c1f0bafc562c88edb16
```

Comment:
Right now the version that is on pypi requires Django < 3.1. They have fixed it but it has not yet been released.